docs/testrunner-toolkit/configuration/playwright.md

```diff
@@ -407,6 +407,20 @@ One or more paths to the playwright test files to run for this suite. Regex valu
 ```yaml
     testMatch: ["**/*.js"]
 ```
+---
+
+#### `numShards`
```

Comment:
```suggestion
### `numShards`
```
I think you mean for this to be at the same level as the other `suite.x` properties, not a sub of `suite.testMatch`?